### DISCUSSION TITLE
Why no `(treesit-auto-add-to-auto-mode-alist 'all)` by default?

### DISCUSSION THREAD

**Discussion by stormwatch** [abougouffa/minemacs#158] [Q&A] [answered]
I was wondering if there is a reason not to add `(treesit-auto-add-to-auto-mode-alist 'all)` to the [`:config`](https://github.com/abougouffa/minemacs/blob/937d19d29704d3a68302e2c69439760fbb81115e/modules/me-prog.el#L23) block in [me-prog.el](https://github.com/abougouffa/minemacs/blob/main/modules/me-prog.el)

**Comment by abougouffa** [comment, marked as answer]
Hi @stormwatch 

Thanks for the suggestion, I wasn't aware of this function, I will add it to my config!
